js/modules/popupRegistration.js

```diff
@@ -0,0 +1,93 @@
+const popupLinks = document.querySelectorAll(".popup-link");
+const registerModalOverlay = document.querySelector(".register__modal_overlay");
+const registerModalContent = registerModalOverlay.querySelector(".register__modal_content");
+const registerModalGratitude = registerModalOverlay.querySelector(".register__modal_gratitude");
+const closeBtn = registerModalOverlay.querySelector(".register__close_button");
+const form = registerModalOverlay.querySelector(".register__form");
+const modalSubmitBtn = registerModalOverlay.querySelector(".modal_form_button");
+const REOPEN_DELAY = 800;
+const SHOW_TIME = 3000;
+const KEYCODE = {
+	ESC: "Escape"
+};
+let isOpen = true;
```

Comment:
1. why do you need `isOpen` ?
2. even if you really need it, it looks very confusing: when modal box is closed `isOpen` is true. When you open modal box `isOpen` is getting false.
3. if you really need it, I think the var name should be `isOpened`